<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_fetch_tweets</class>
 <widget class="QWidget" name="widget_fetch_tweets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fetch Tweets</string>
  </property>
  <property name="statusTip">
   <string extracomment="search"/>
  </property>
  <property name="whatsThis">
   <string extracomment="search"/>
  </property>
  <widget class="QWidget" name="widget_querry_container" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="button_fetch_tweets">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>10</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>search</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>mg.png</normaloff>mg.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_querry">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> qproperty-frame: false </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_save_tweets">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>148</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>save to csv</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Save.png</normaloff>Save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="button_stop_fetching">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>148</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>stop fetching</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>stop.png</normaloff>stop.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>152</y>
     <width>97</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tweet count:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_tweet_count">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>151</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>widget_querry_container</zorder>
  <zorder>button_save_tweets</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
